andela-mabdussalam/restaurant-app
# Feat/update landing page

#### What does this PR do?
This PR changes the folder structure.

#### Description of Task to be completed?
- Moving all app related folders into the `app` folder.

#### How should this be manually tested?
- Clone the repo
- Checkout to the develop branch
- Check folder structure
